src/unix/notbsd/linux/mod.rs

```diff
@@ -185,6 +185,44 @@ s! {
     }
 }
 
+f! {
+    // FIXME(peterhj, 20160204): should not have pub visibility.
```

Comment:
Could these definitions just be inlined below? It's generally ok to have a little more repetition to avoid exposing too many internal details.